libafl_frida/src/alloc.rs

```diff
@@ -587,6 +589,14 @@ impl Allocator {
             &mut |range: &RangeDetails| -> bool {
                 let start = range.memory_range().base_address().0 as usize;
                 let end = start + range.memory_range().size();
+                log::trace!("New range: {:#x}-{:#x}", start, end);
+
+                #[cfg(target_vendor = "apple")]
+                if start >= 0x600000000000 {
+                    //this is the MALLOC_NANO region. There is no point in poisoning this as we replace malloc.
```

Comment:
what do you mean poisoning? This loop doesn't poison anything....

Reply:
This is referring to the fact that on ios devices `MALLOC_NANO` does not need to be poisoned/unpoisoned as we replace malloc. As a result, we don't need to keep track of it and unpoison it later in `unpoison_all_existing_memory`. I'll clarify that comment